lib/tasks/result_monitor.rake

```diff
@@ -76,7 +72,8 @@ task "result_monitor", [:duration] => :environment do |_t, args|
   # spawn a new finite duration process every 60 minutes
   respawn_interval = 60 * 60
   # rate-limit status updates
-  checks_per_minute = 4.0
+  cloud_env = ["prod", "staging"].include?(Rails.env)
+  checks_per_minute = cloud_env ? 4.0 : 0.2
```

Comment:
4 checks per minute makes laptop very hot!

1 check every 5 minutes is barely acceptable... due to the result monitor inefficiency

this wasn't noticeable when the environment was totally broken, but since the rest of this pull request fixes the environment, heat becomes a concern